server/traces/otel_http_converter.go

```diff
@@ -23,10 +24,17 @@ type httpScopeSpans struct {
 
 type httpSpan struct {
 	v1.Span
-	SpanId       string               `json:"spanId"`
-	ParentSpanId string               `json:"parentSpanId"`
-	Kind         string               `json:"kind"`
-	Attributes   []*httpSpanAttribute `json:"attributes"`
+	SpanId            string               `json:"spanId"`
+	ParentSpanId      string               `json:"parentSpanId"`
+	Kind              string               `json:"kind"`
+	StartTimeUnixNano string               `json:"startTimeUnixNano"`
+	EndTimeUnixNano   string               `json:"endTimeUnixNano"`
+	Attributes        []*httpSpanAttribute `json:"attributes"`
+	Status            *httpSpanStatus      `json:"status"`
+}
+
+type httpSpanStatus struct {
+	Code string `json:"code"`
```

Comment:
We were using the v1.Span Status Code field which is an `int`. Thus it would fail to convert the status "STATUS_OK" into an integer.